variables.tf

```
@@ -361,6 +361,33 @@ variable "advanced_threat_protection_enabled" {
   nullable    = false
 }
 
+variable "azure_files_authentication_directory_type" {
+  description = "Determines if azure_files_authentication should be enabled. Possible values are AD, AADS or AADKERB"
```

I suggest that we update description according to [this document](https://learn.microsoft.com/en-us/azure/storage/files/storage-files-active-directory-overview):

```suggestion
  description = "The type of directory service to use for Azure Files identity-based authentication. Value must be \"AD\" (Active Directory Domain Services), \"AADDS\" (Microsoft Entra Domain Services), \"AADKERB\" (Microsoft Entra Kerberos) or null (identity-based authentication disabled)."
```

- Clarify the purpose of this variable (specify the type of directory service to use for Azure Files identity-based authentication).
- Clarify that `null` is a supported value.
- Clarify **what** the supported values mean.

Could probably throw a link to that document in right above the `azure_files_authentication` block (`main.tf`, line 43), for future reference (similar to [this](https://github.com/equinor/terraform-azurerm-storage/blob/0ff30994f1df92916d6e21c94b020d0703964225/main.tf#L175)).